### PR TITLE
LPS-75116 Apply gradle formatSource

### PR DESCRIPTION
**Pull request by clovisdasilvaneto**

### CODE REVIEW

**Comment by rafaprax** [comment]
ci:retest

File: dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_field_settings_sidebar.js [hunk 1]
```diff
@@ -68,6 +69,49 @@ AUI.add(
...
+						var instance = this;
+
+						var fieldNode = event.currentTarget;
+						var type = fieldNode.getData('name');
+
+						var fieldType = FieldTypes.get(type);
+						var FormBuilder = instance.get('builder');
+
+						var field = FormBuilder.createField(fieldType);
+						var prevField = instance.get('field');
```
Comment (rafaprax) [inline dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_field_settings_sidebar.js]: @clovisdasilvaneto rename to previousField

File: dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_field_settings_sidebar.js [hunk 2]
```diff
@@ -268,6 +315,39 @@ AUI.add(
...
+
+						FormBuilderUtil.visitLayout(
+							newSettingsContext.pages,
+							function(settingsFormFieldContext) {
+								var fieldLocalizable = settingsFormFieldContext.localizable;
+								var fieldName = settingsFormFieldContext.fieldName;
+
+								FormBuilderUtil.visitLayout(
+									previousSettingsContext.pages,
+									function(prevSettingsFormFieldContext) {
```
Comment (rafaprax) [inline dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_field_settings_sidebar.js]: @clovisdasilvaneto rename to previousSettingsFormFieldContext

File: dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_field_settings_sidebar.js [hunk 3]
```diff
@@ -323,6 +403,27 @@ AUI.add(
 						}
 
 						boundingBox.addClass('loading-data');
+					},
+
+					_updateField: function(prevField, field, column) {
```
Comment (rafaprax) [inline dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_field_settings_sidebar.js]: @clovisdasilvaneto rename to previousField

File: dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/templates/sidebar.soy [hunk 4]
```diff
@@ -1,12 +1,48 @@
...
+{template .header}
+{@param closeButtonIcon: html}
+{@param toolbarButtonIcon: html}
+{@param? toolbarTemplateContext: [options: list<[buttonClass: string, handler: string, label: string]>]}
+{@param? type: string}
+
+<div class="sidebar-header">
+	<div class="sidebar-section-flex">
+		<div class="flex-col flex-col-expand">
+			{if $type}
```
Comment (rafaprax) [inline dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/templates/sidebar.soy]: @clovisdasilvaneto Add one more tab to `delcall ` statements

**Comment by clovisdasilvaneto** [comment]
Hey @rafaprax the modifications above were completed, can you review again?

**Comment by pedroqueiroz94** [comment]
ci:retest

**Comment by rafaprax** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by rafaprax** [comment]
@clovisdasilvaneto real errors here 😢

**Pull request closed without merging**